pkg/flowaggregator/certificate.go

```diff
@@ -138,18 +139,34 @@ func generateCertKey(caCert *x509.Certificate, caKey *rsa.PrivateKey, isServer b
 func syncCAAndClientCert(caCert, clientCert, clientKey []byte, k8sClient kubernetes.Interface) error {
 	klog.Info("Syncing CA certificate, client certificate and client key with ConfigMap")
 	caConfigMap, err := k8sClient.CoreV1().ConfigMaps(CAConfigMapNamespace).Get(context.TODO(), CAConfigMapName, metav1.GetOptions{})
+	exists := true
 	if err != nil {
-		return fmt.Errorf("error getting ConfigMap %s: %v", CAConfigMapName, err)
+		if !errors.IsNotFound(err) {
+			return fmt.Errorf("error getting ConfigMap %s: %v", CAConfigMapName, err)
+		}
+		exists = false
+		caConfigMap = &v1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
```

Comment:
Please add the label as the previous one in manifest yaml, in case we need a way to filter all antrea resources later.